modules/op-configuring-pipeline-resolvers.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-configuring-pipeline-resolvers_{context}"]
+= Configuring pipeline resolvers
+
+:FeatureName: Resolvers
+include::snippets/technology-preview.adoc[]
+
+You can configure pipeline resolvers in the `TektonConfig` CR. You can enable or disable these pipeline resolvers:`enable-bundles-resolver`, `enable-cluster-resolver`, `enable-git-resolver`, and `enable-hub-resolver`.
```

Comment:
"custom resource (CR)" - since it's the first instance of CR in the module it should be explained